org.eclipse.jdt.junit/src/org/eclipse/jdt/junit/wizards/NewTestSuiteWizardPage.java

```diff
@@ -758,14 +758,16 @@ private void internalSetJUnit4(boolean isJUnit4) {
 	}
 
 	@Override
-	protected String constructCUContent(ICompilationUnit cu, String typeContent, String lineDelimiter) throws CoreException {
```

Comment:
you must not change the Signature of methods that are API. or you get 
`The method org.eclipse.jdt.junit.wizards.NewTestSuiteWizardPage.constructCUContent(ICompilationUnit, String, String) has been removed`
You can however add and additional method and make sure that the old API still works.